core/src/main/java/bisq/core/dao/node/parser/TxOutputParser.java

```diff
@@ -285,14 +285,15 @@ private boolean isBtcOutputOfBurnFeeTx(TempTxOutput tempTxOutput) {
                         // We always have the BSQ change before the burned BSQ output if both are present.
                         checkArgument(optionalOpReturnIndex.isPresent());
                         if (index != optionalOpReturnIndex.get() - 1) {
-                            return false;
+                            break;
                         }
 
                         // Without checking the fee we would not be able to distinguish between 2 structurally same transactions, one
                         // where the output is burned BSQ and one where it is a BSQ change output.
                         long blindVoteFee = daoStateService.getParamValueAsCoin(Param.BLIND_VOTE_FEE, tempTxOutput.getBlockHeight()).value;
                         return availableInputValue == blindVoteFee;
                     }
+                    break;
```

Comment:
If the semantic changes in this file are reverted, take care not to revert adding this `break`.

Reply:
I'm doing a revert of the whole PR. There was an easy to use button to do the revert.

Best make a new PR with the good changes.